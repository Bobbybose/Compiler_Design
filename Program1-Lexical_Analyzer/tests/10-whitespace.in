"Before space" "After space"
"Before tab"    "After tab"
"Before newline"
"After newline"