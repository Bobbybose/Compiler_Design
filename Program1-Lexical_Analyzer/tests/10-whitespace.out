string-literal: "Before space": 1, 1
string-literal: "After space": 1, 16
string-literal: "Before tab": 2, 1
string-literal: "After tab": 2, 17
string-literal: "Before newline": 3, 1
string-literal: "After newline": 4, 1